### DISCUSSION TITLE
Add "other" category for Energy Devices Detail Graph

### DISCUSSION THREAD

**Discussion by alexpomsft** [home-assistant/frontend#20079] [Dashboard]
The `energy-devices-detail-graph` view is awesome, but it would be great if we could also graph the "unknown" or "other" block of energy usage, defined as total energy used minus the amount which is shown as specific devices. I think if we had this feature we could completely collapse the two existing energy graphs into one, which is simply configurable on how many devices it shows. Perhaps another way of thinking about this is to enable `energy-usage-graph` to show x number of specific devices.

**Comment by gnu2000** [comment]
I'd love to see this. The individual devices list works great, but it would be nice to see the wider context on the stacked graph.
